### ISSUE TITLE
About Fold and Reaction dataset in torchdrug

### ISSUE THREAD

**Issue by Kirito-Ausna** [DeepGraphLearning/GearNet#14]
Hi, sorry for bothering you about Torch drug API. I want to reproduce your results on Fold Classification and Reaction. However, I find that the Fold dataset in torchdrug/datasets/fold/Fold doesn't contain protein structure data? It basically contains Sequences and Labels. According to my understanding of GearNet, it's a structure-based method and the pretrain tasks are also structure-based. So, I am confused about current situation.
Besides, I can't find the reaction dataset in torchdrug. Could you please tell me which dataset you used and the training config like EC and GO.
 
Sorry again for adding to your trouble. Thank you for open sourcing such a great work.

**Comment by Oxer11** [comment]
Hi! Thanks for your question!

I haven't made the codes for Fold and Reaction public yet. This will be done when the paper is accepted. If you want to reproduce the results on these two datasets, the datasets are from [the repo of IEConv](https://github.com/phermosilla/IEConv_proteins/tree/master/Datasets/data) though the implementation and hyperparameters of our model may be slightly different here.

**Comment by Kirito-Ausna** [comment]
Thank you for your kind reply, good luck！
